api/v1/helper.go

```diff
@@ -481,7 +481,7 @@ func (p *SriovNetworkNodePolicy) generateVfGroup(iface *InterfaceExt) (*VfGroup,
 	var rngStart, rngEnd int
 	found := false
 	for _, selector := range p.Spec.NicSelector.PfNames {
```

Comment:
Ranges are still defined on `PfNames` field only. 
You can verify it by adding unit tests like  https://github.com/k8snetworkplumbingwg/sriov-network-operator/commit/a92b76433a5debe9749e82a66ebcb35536b2453e .

Reply:
you are right!

I think supporting rage in the root device will be much harder than I was expecting so I will first fix the webhook issue and than we can take to the community to talk about the range support for rootDevices.

let me update the PR